dotCMS/src/main/java/com/liferay/portal/ejb/CompanyPersistence.java

```diff
@@ -82,7 +83,7 @@
 				session = openSession();
 
 				if (company.isNew()) {
-					CompanyHBM companyHBM = new CompanyHBM(company.getCompanyId(),
+					final CompanyHBM companyHBM = new CompanyHBM(company.getCompanyId(),

```

Comment:
I did consider that.. and I  concluded it wasn't required.. since we're only saving one entity. 
though I had to try many different things before I could make this work. I'll have it added.

Reply:
This is the same pattern used in Users (which BTW is where I had initially focused my test efforts). I completely missed this one.

Reply:
I'll have the test method added

Reply:
Requested changes.. done!